apps/concierge_site/lib/helpers/mail_helper.ex

```diff
@@ -54,8 +54,8 @@ defmodule ConciergeSite.Helpers.MailHelper do
     Helpers.email_opened_url(
       ConciergeSite.Endpoint,
       :notification,
-      alert_id: alert_id,
-      notification_id: notification_id
+      alert_id,
+      notification_id
```

Comment:
I think this all fits on one line now. `mix format` never removes line breaks, even if doing so would allow fitting a statement on fewer lines.